Bugzilla/User.pm

```diff
@@ -2228,10 +2250,8 @@ our %names_to_events = (
     'resolution'              => EVT_OPENED_CLOSED,
     'keywords'                => EVT_KEYWORD,
     'cc'                      => EVT_CC,
-    'bug_severity'            => EVT_PROJ_MANAGEMENT,
     'priority'                => EVT_PROJ_MANAGEMENT,
     'bug_status'              => EVT_PROJ_MANAGEMENT,
-    'target_milestone'        => EVT_PROJ_MANAGEMENT,
```

Comment:
these changes don't appear to be related to this bug

Reply:
woops, those were accidental.